doc/changes.rst

```diff
@@ -7,7 +7,6 @@ Change Log
 
 *
 
-
```

Comment:
I'm sorry, but now it looks like you deleted all the notes that were already under 1.9.3.  I strongly suggest you revert to the master version, then add in your *one line* about the change under 1.9.3 (unreleased).